src/components/About.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import AboutImg from '../assets/about-image.svg'
+
+function About() {
+  return (
+    <div className='w-full my-32 ' > 
+        <div className=' flex justify-center flex-col bg-lightBlue'>
+            <div className='text-center flex-col'>
+               <h2 className='text-4xl font-bold'>About</h2> 
+               <p className='text-3xl py-3 text-gray-500'>In quo quaerimus, non emolumento aliquo, sed ipsius honestatis decore laudandis, id est et quas molestias excepturi sint, obcaecati cupiditate non numquam eius modi tempora incidunt, ut earum rerum hic tenetur a natura ipsa natura incorrupte atque integre iudicante itaque aiunt hanc quasi involuta aperiri.</p> 
+            </div>
+             <button className='bg-darkBlue text-[#00df9a] w-[200px] rounded-md font-medium my-6 mx-auto md:mx-0 py-3' >Learn more</button>
+        </div>
+        <div>
+            <div className= 'grid grid-rows-2 columns-2 '>
```

Comment:
this div's width can be:   w-1/2  so other picture div can be w-1/2 

1. you can use seperate div for each topic 
2. you may change the div's width
3. adding mini symbol star for each topic